File: app/src/main/java/com/illidant/easykanzicapstone/domain/model/Kanji.java
package com.illidant.easykanzicapstone.domain.model;


public class Kanji {
    private String Title;
    private String Category;
    private String Description;
    private String Thumbnail;

    public Kanji() {
    }

    public Kanji(String title, String category, String description, String thumbnail) {
        Title = title;
        Category = category;
        Description = description;
        Thumbnail = thumbnail;
    }

    public String getTitle() {
        return Title;
    }

    public void setTitle(String title) {
        Title = title;
    }

    public String getCategory() {
        return Category;
    }

    public void setCategory(String category) {
        Category = category;
    }

    public String getDescription() {
        return Description;
    }

    public void setDescription(String description) {
        Description = description;
    }

    public String getThumbnail() {
        return Thumbnail;
    }

    public void setThumbnail(String thumbnail) {
        Thumbnail = thumbnail;
    }
}

File: app/src/main/java/com/illidant/easykanzicapstone/domain/model/Kanji.kt
package com.illidant.easykanzicapstone.domain.model

import com.google.gson.annotations.SerializedName

data class Kanji(
    @SerializedName("id")
    val id: Int,
    @SerializedName("kanji")
    val kanji: String,
    @SerializedName("sino_vietnamese")
    val sino_vietnamese: String,
    @SerializedName("meaning")
    val meaning: String,
    @SerializedName("onyomi")
    val onyomi: String,
    @SerializedName("on_furigana")
    val on_furigana: String,
    @SerializedName("kunyomi")
    val kunyomi: String,
    @SerializedName("kun_furigana")
    val kun_furigana: String
)
